src/Assert.php

```diff
@@ -107,6 +107,25 @@ public static function integerish($value, $message = '')
         }
     }
 
+    /**
+     * @psalm-pure
+     * @psalm-assert positive-int $value
```

Comment:
couldn't find positive-int in the psalm docs anywhere... can you point me to where its documented?

Reply:
https://github.com/vimeo/psalm/releases/tag/3.13.0

Reply:
Thx, added a docs PR https://github.com/vimeo/psalm/pull/4078